packages/datatrak-web/src/views/SurveySelectPage.tsx

```diff
@@ -2,21 +2,36 @@
  * Tupaia
  *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router';
 import styled from 'styled-components';
 import { DialogActions, Paper, Typography } from '@material-ui/core';
 import { Description, FolderOpenTwoTone } from '@material-ui/icons';
-import { SpinningLoader } from '@tupaia/ui-components';
-import { useSurveys } from '../api/queries';
+import { SpinningLoader, Select as BaseSelect } from '@tupaia/ui-components';
+import { useEditUser } from '../api/mutations';
 import { SelectList, ListItemType, Button } from '../components';
-import { Survey } from '../types';
+import { Entity, Survey } from '../types';
+import { useUserCountries } from '../utils';
+import { useSurveys } from '../api/queries';
 
 const Container = styled(Paper).attrs({
   variant: 'outlined',
 })`
   width: 48rem;
   display: flex;
   flex-direction: column;
```

Comment:
I'm not sure how to best fix it, but the paper moves around a bit when I select a country because the length of the list changes. Maybe this should be justified to the top and there could be a min height?